tests/unit/test-util/test.adb.js

```diff
@@ -935,6 +935,52 @@ describe('utils/adb', () => {
       );
     });
 
+    async function testReferenceBrowserApkComponent(
+      firefoxApkComponent?: string, expectedApkComponent: string
+    ) {
+      const adb = getFakeADBKit({
+        adbClient: {
+          startActivity: sinon.spy(() => Promise.resolve()),
+        },
+        adbkitUtil: {
+          readAll: sinon.spy(() => Promise.resolve(Buffer.from('\n'))),
+        },
+      });
+      const adbUtils = new ADBUtils({adb});
+      const apkName = 'org.mozilla.reference.browser';
+      const component = `${apkName}/${apkName}.${expectedApkComponent}`;
+      const promise = adbUtils.startFirefoxAPK(
+        'device1',
+        apkName,
+        firefoxApkComponent, // firefoxApkComponent
```

Comment:
Nit, no need for the comment now that we are passing an explicit variable with the exact same name.

Reply:
Done 🚀